plugins/catalog/src/alpha/entityCards.tsx

```diff
@@ -92,7 +95,43 @@ export const catalogHasSystemsEntityCard = createEntityCardExtension({
     ),
 });
 
+export const catalogOrphanWarningEntityCard = createEntityCardExtension({
+  name: 'orphan-warning',
+  disabled: false,
+  fullWidth: true,
```

Comment:
Created this option to inform the page that this card should fill the entire row.
Also open to suggestions 💡

Reply:
We spoke a bit elsewhere and I argued that maybe this should be treated as a new "admonitions" feature separate from cards both conceptually and layout-wise. Ideally they'd even show everywhere and not just on the overview tab. We'll chat about it on Monday.

Reply:
As we discussed, we decided to add manually the entity warning components to the overview tab for now and not change the entity page design for now.